docs/man-pages/condor_transform_ads.rst

```diff
@@ -10,23 +10,27 @@ Synopsis
 
 **condor_transform_ads** [**-help [rules]** ]
 
-**condor_transform_ads** **-rules** *rules-file*
+**condor_transform_ads** [**-rules** *rules-file*]
+[**-jobtransforms** *name-list*]
 [**-in[:<form>]  ** *infile*] [**-out[:<form>[,
 nosort]]  ** *outfile*] [*<key>=<value>* ] [**-long** ] [**-json** ]
 [**-xml** ] [**-verbose** ] [**-terse** ] [**-debug** ]
 [**-unit-test** ] [**-testing** ] [**-convertoldroutes** ] [*infile1
 ...infileN* ]
 
-Note that exactly one rules file, and at least one input file, must be
-specified. If no output file is specified, output will be written to
+Note that one or more transform must be specified in the form of a rules
+file or a ``JOB_TRANSFORM_`` name and at least one input file must be
+specified. Transforms will be applied in the order they are given on the command
+line.  If a complex TRANSFORM statement is used it must be at the end of the last
```

Comment:
What is "a complex TRANSFORM statement"?  (If it's defined elsewhere in the manual, that's fine, but I couldn't find it.)

Reply:
I think the code calls these "iterating" transforms?

Reply:
yeah. iterating transforms

Reply:
I don't actually know what those are, but if people who write transforms do, maybe just say that instead?